common/rpc/interceptor/telemetry.go

```diff
@@ -52,7 +52,7 @@ import (
 type (
 	metricsContextKey struct{}
 
-	TelemetryInterceptor struct {
```

Comment:
Go style guide suggests avoiding repeating the package name for exported symbols since the package name will always be visible. I don't think we use this struct inside this package, so it will always be written as `interceptor.Telemetry`, which I think is still clear.

Reply:
I'm familiar with the style guide. Still feels like this is losing information.

If it was named `telemetry.Interceptor` I'd be totally on board.
In any case, I suggest that if we want to rename the interceptors we should do it in a separate PR and apply consistently to all of them.